File: projetoES3/src/model/domain/Pais.java
package model.domain;
public class Pais extends EntidadeDominio {

	private String siglaPais;
	private String descricaoPais;


	public Pais(String sigla, String descricao) {
		this.siglaPais = sigla;
		this.descricaoPais = descricao;
	}
	
	public Pais() {	
	}
	
	public Pais(String descricao) {	
		this.descricaoPais = descricao;
	}

	public String getSigla() {
		return siglaPais;
	}

	public void setSigla(String sigla) {
		this.siglaPais = sigla;
	}

	public String getDescricao() {
		return descricaoPais;
	}

	public void setDescricao(String descricao) {
		this.descricaoPais = descricao;
	}
}

File: typeScript/src/model/domain/Pais.ts
export class Pais {
    constructor(
        private id: number,
        private siglaPais: string,
        private descricaoPais: string
    ) {
    }

    public getId(): number {
        return this.id;
    }

    public setId(id: number): void {
        this.id = id;
    }

    public Pais(descricao: string) {
        this.descricaoPais = descricao;
    }

    public getSigla(): string {
        return this.siglaPais;
    }

    public setSigla(sigla: string): void {
        this.siglaPais = sigla;
    }

    public getDescricao(): string {
        return this.descricaoPais;
    }

    public setDescricao(descricao: string): void {
        this.descricaoPais = descricao;
    }
}


